app/assets/stylesheets/base/_fonts.scss

```diff
@@ -1,221 +1,477 @@
 // This file contains the base fonts for the application.
 
-$font-cdn-base: 'https://cdn.jsdelivr.net/font-lato/2.0/Lato';
+$full-set-cdn: 'https://cdn.jsdelivr.net/font-lato/2.0/Lato';
+$non-latin-range:
```

Comment:
Oh snap, you manually figured this out? Wow 🥇

Reply:
In my own weird way, going through each character is like Tibetan monks making mandala sand art.

Reply:
Do you anticipate ever having to update this list?

Reply:
By that time we will want to have automated deployment scripts to split the font by language. This was just a low-hanging fruit in that Lato officially released Latin vs full set versions of its font.